scripts/abi_bindings.sh

```diff
@@ -64,26 +64,66 @@ function convertToLower() {
     fi
 }
 
+remove_matching_string() {
```

Comment:
Could you add a small doc comment for this bash function here?

Reply:
Addressed in https://github.com/ava-labs/teleporter/pull/626